packages/gatsby-source-contentful/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "gatsby-source-contentful",
   "description": "Gatsby source plugin for building websites using the Contentful CMS as a data source",
-  "version": "4.6.0",
```

Comment:
chose this for zero chance of future collision - which is impossible sense we are past EoL on these versions but whatever